rules/busybox.bzl

```diff
@@ -641,14 +645,33 @@ def _compile(
     )
     args.add("--output", out_file)
 
+    optional_outputs = []
+    if out_class_jar:
+        args.add("--classJarOutput", out_class_jar)
```

Comment:
This is throwing the following error on internal tests:

```
Error parsing command line: Unrecognized option: --classJarOutput
```

Which is kind of surprising. I'll look into this a bit.

Reply:
You need something like this, I think:

```
    optional_outputs = []
    optional_args = []
    if out_class_jar:
        optional_args.add("--classJarOutput", out_class_jar)
        optional_outputs.append(out_class_jar)
    if out_r_txt:
        optional_args.add("--rTxtOut", out_r_txt)
        optional_outputs.append(out_r_txt)

    if len(optional_outputs) > 1:
        args.extend(["--"], optional_args)
```